README.md

```diff
@@ -151,6 +164,28 @@ Pseudo-R^2
 ---------------------------
 explained_deviance     0.46
 ```
+
+Like for other GAMs, we can simulate from the posterior of the coefficients and smoothing parameters:
+
+
+```python
+fig, axs = plt.subplots(1, 3)
+titles = ['student', 'balance', 'income']
+
+for i, ax in enumerate(axs):
+    pdep = gam.partial_dependence(XX, feature=i+1)
+    for sample_vector in samples:
```

Comment:
@cbrummitt im just seeing this logistic regression chart for the first time. very cool graphic.
can you explain it to me? how come all the charts look so similar?

also, can you include a snippet that generated these samples?

Reply:
Good catch. I wasn't thinking clearly enough. You're totally right: the vertical positions are exactly the same; only the horizontal scale is different in the three panels. I'd have to do a variant of `partial_dependence` that draws random samples, which doesn't seem worth it. I just removed this addition to the README. Just one illustration of `sample` (namely, `pygam_mcycle_data_linear_sample_from_posterior.png`) should be enough